js/plugin/RouteLoaderConverter.js

```diff
@@ -328,7 +329,41 @@ BR.routeLoader = function (map, layersControl, routing, pois) {
                 this.getSimplifiedCoords(this._options.simplifyTolerance)
             );
 
+            // use now the middle point between the "simplified" waypoint and the "previous" trkpt in the origin track
+
+            var trackPt_b = turf.cleanCoords(this._trackPoints.geometry);
+            var trackPt_bis = L.GeoJSON.coordsToLatLngs(trackPt_b.coordinates);
```

Comment:
Please avoid cryptic variable names with single letters or uncommon abbreviations (what does `b` or `bis` mean?) or just numbering like `buf1`/`2`/`3`, and use [camelCase](https://en.wikipedia.org/wiki/Camel_case) instead of underscore (`_`), e.g. `trackPt_bis` could be `trackLatLngs`